src/dbnode/encoding/tile/series_frame_summary.go

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) 2020 Uber Technologies, Inc.
```

Comment:
This file was not a part of the original PR. Are we are using it somewhere?

Reply:
This was one of the later PRs that didn't get merged into that combined one; it provides a running summary for the current tile to calculate a bunch of aggregated values